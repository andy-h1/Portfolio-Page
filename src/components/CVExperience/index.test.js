/* eslint-disable max-len */

import { render } from "@testing-library/react";
import deliveroo from "../../tokens/CompanyLogos/deliveroo.png";
import { CVExperience } from ".";

const props = {
  dates: "Nov 2018 - Present",
  description: [
    "I joined the Deliveroo recruitment team focused on hiring Product Managers, Product Marketing Managers and Software Engineers. When I joined we were around 20+ in the team and now have grown to over 60+ internationally",
    "I have been involved in hiring across different levels of seniority but mostly focused on Senior/L6+.",
    "Aside from recruiting for the Engineering team at Deliveroo, I have been heavily involved in onboarding, training and building out our recruitment team.",
  ],
  image: deliveroo,
  imageAlt: "deliveroo",
  imageHeight: "auto",
  imageWidth: "60px",
  jobTitle: "Tech Recruiter",
  website: "https://deliveroo.co.uk/",
};

describe("CV Experience component", () => {
  it("should render", () => {
    const { container } = render(<CVExperience {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
