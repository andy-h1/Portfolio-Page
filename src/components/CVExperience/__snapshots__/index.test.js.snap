// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CV Experience component should render 1`] = `
<li
  class="sc-AxjAm dPRGBU"
>
  <a
    href="https://deliveroo.co.uk/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="deliveroo"
      height="auto"
      src="deliveroo.png"
      width="60px"
    />
  </a>
  <h3
    class="sc-AxirZ fDOJrF"
  >
    Tech Recruiter
  </h3>
  <h3
    class="sc-AxiKw gyIrqT"
  >
    Nov 2018 - Present
  </h3>
  <p
    class="sc-AxhCb ldYzEm"
  >
    I joined the Deliveroo recruitment team focused on hiring Product Managers, Product Marketing Managers and Software Engineers. When I joined we were around 20+ in the team and now have grown to over 60+ internationally
  </p>
  <p
    class="sc-AxhCb ldYzEm"
  >
    I have been involved in hiring across different levels of seniority but mostly focused on Senior/L6+.
  </p>
  <p
    class="sc-AxhCb ldYzEm"
  >
    Aside from recruiting for the Engineering team at Deliveroo, I have been heavily involved in onboarding, training and building out our recruitment team.
  </p>
</li>
`;
